app/src/pages/inside/testsPage/testsPage.jsx

```diff
@@ -108,6 +109,7 @@ export class TestsPage extends Component {
     } = this.props;
     return (
       <PageLayout>
+        <HistoryLine activePage={activePage} pageSize={pageSize} sortingColumn={sortingColumn} />
```

Comment:
You don't use this props inside of component. Remove it please

Reply:
Fixed